tedana/resources/decision_trees/minimal.json

```diff
@@ -29,10 +29,12 @@
                 "decide_comps": "all"
             },
             "kwargs": {
-                "log_extra_report": "",
                 "clear_classification_tags": true,
-                "dont_warn_reclassify": true
-            }
+                "dont_warn_reclassify": true,
+                "log_extra_info": "",
+                "log_extra_report": "All components are initially labeled as 'unclassified'."
+            },
+            "_comment": "This step prepares the components to be classified by the decision tree."
```

Comment:
The info log will already output:
"Set all to unclassified"
"component classification tags are cleared"

Just to it's clear in the json file, maybe change `_comment` to "All components are set to unclassified to initialize the tree" and remove `log_extra_report`